watcher/watcher.go

```diff
@@ -196,10 +186,24 @@ func (w *Watcher) AutoScale() {
 	}
 
 	l := logger.WithFields(logrus.Fields{
-		"roomsByStatus": roomCountByStatus,
-		"state":         scheduler.State,
+		"ready":       roomCountByStatus.Ready,
+		"creating":    roomCountByStatus.Creating,
+		"occupied":    roomCountByStatus.Occupied,
+		"terminating": roomCountByStatus.Terminating,
+		"state":       scheduler.State,
 	})
 
+	err = controller.CreateNamespaceIfNecessary(
```

Comment:
Ok this makes sense. We'll just create the namespace if the scheduler still exists in the db.